Scripts/Mac/Tracing/dtruss

```diff
@@ -0,0 +1,1271 @@
+#!/bin/sh
+# #!/usr/bin/sh
+#
+# dtruss - print process system call time details.
+#          Written using DTrace (Solaris 10 3/05).
```

Comment:
Where is this coming from?

Reply:
See my comment on licensing https://github.com/Microsoft/VFSForGit/pull/737/commits/a5176525b4b0a84e686a8df5a49ddb4137fbc861#r254600223